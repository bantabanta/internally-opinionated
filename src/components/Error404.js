import { Link } from "react-router-dom";

const Error404 = () => {
  return (
    <div className="content-wrapper poll">
      <h1>Uh-oh 404</h1>
      <h3>The internally opinionated are lost...</h3>
      <h4>This page isn't available</h4>
      <p>
        The link may be broken, or the page may have been removed. Check to see
        if the link you're trying to open is correct.
      </p>
      <Link to="/">
        <button>Return to Home</button>
      </Link>
    </div>
  );
};

export default Error404;
