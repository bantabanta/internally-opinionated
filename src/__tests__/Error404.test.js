import { render } from "@testing-library/react";
import * as React from "react";
import Error404 from "../components/Error404";
import { BrowserRouter as Router } from "react-router-dom";

describe("Dashboard", () => {
  it("will match snapshot", () => {
    const view = render(
      <Router>
        <Error404 />
      </Router>
    );
    expect(view).toMatchSnapshot();
  });
});
