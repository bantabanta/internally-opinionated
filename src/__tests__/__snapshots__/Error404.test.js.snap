// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard will match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content-wrapper"
      >
        <div
          class="center"
        >
          <h1>
            Uh-oh 404
          </h1>
          <h2
            class="text-standout"
          >
            The internally opinionated are lost!
          </h2>
          <p>
            Sorry, this page isn't available...
          </p>
          <p>
            The link may be broken, or the page may have been removed.
          </p>
          <a
            href="/"
          >
            <button>
              Return to Home
            </button>
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content-wrapper"
    >
      <div
        class="center"
      >
        <h1>
          Uh-oh 404
        </h1>
        <h2
          class="text-standout"
        >
          The internally opinionated are lost!
        </h2>
        <p>
          Sorry, this page isn't available...
        </p>
        <p>
          The link may be broken, or the page may have been removed.
        </p>
        <a
          href="/"
        >
          <button>
            Return to Home
          </button>
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
